core/src/repair_service.rs

```diff
@@ -932,6 +981,11 @@ mod test {
         Blockstore::destroy(&blockstore_path).expect("Expected successful database destruction");
     }
 
+    fn post_shred_deferment_timestamp() -> u64 {
```

Comment:
Should this be test only function and shared with crate?

Reply:
shared `post_shred_deferment_timestamp` to crate and deduped the function.